### DISCUSSION TITLE
Question on Auto Execute Macro

### DISCUSSION THREAD

**Discussion by discodotgg** [vigoren/foundryvtt-simple-calendar#487] [Q&A]
Hi,

Maybe I'm not figuring this out, but when creating a new note, when I select the Auto Execute Macro, the drop down list shows my macros there, but it doesn't matter which macro I click, the selection always stays "No Macro". Am I missing a step?

![image](https://user-images.githubusercontent.com/131381550/233438997-f0b32ffe-5720-4175-acb8-9d4b4d3ad78b.png)

Thanks,

**Comment by vigoren** [comment]
Hey sorry about the long delay in getting back to you about this (had a couple months of medical issues).

There was a bug that was preventing this from working that should be fixed in version 2.2.0 of Simple Calendar.
